app/lib/image_pipeline_service.rb

```diff
@@ -23,25 +45,45 @@ class ImagePipelineService
   # * *raises*
   #   - (ArgumentError) => if requested parameters do not validate
   def self.run_render_expression_arrays_job(study, cluster_file, matrix_file, user: nil)
-    raise ArgumentError, 'invalid study' unless study.is_a?(Study)
-
+    validate_study(study)
     requested_user = user || study.user
     params_object = create_expression_parameters_object(cluster_file, matrix_file)
+    submit_job(study:, cluster_file:, requested_user:, params_object:)
+  end
+
+  # generic job submission handler
+  #
+  # * *params*
+  #   - +study+          (Study) => study to launch job for
+  #   - +cluster_file+    (StudyFile) => clustering file to use as source for cell names
+  #   - +requested_user+ (User) => associated user (for email notifications)
+  #   - +params_object+  (Multiple) => parameters object, e.g. ImagePipelineParameters
```

Comment:
What does `Multiple` mean here?  I'm unfamiliar with it as type annotation, is it perhaps like `any` in TypeScript?  Naively, I'd thought I'd see something like `Hash` here.